src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -0,0 +1,209 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['appconfig'] = """
+    type: group
+    short-summary: Manage App Configurations.
+    """
+
+helps['appconfig create'] = """
+    type: command
+    short-summary: Create an App Configuration.
+    examples:
+        - name: Create an App Configuration with name, location and resource group.
+          text:
+            az appconfig create -g MyResourceGroup -n MyAppConfiguration -l westus
+    """
+
+helps['appconfig delete'] = """
+    type: command
+    short-summary: Delete an App Configuration.
+    examples:
+        - name: Delete an App Configuration under resource group
+          text:
+            az appconfig delete -g MyResourceGroup -n MyAppConfiguration
+    """
+
+helps['appconfig update'] = """
+    type: command
+    short-summary: Update an App Configuration.
+    examples:
+        - name: Update tags of an App Configuration
+          text:
+            az appconfig update -g MyResourceGroup -n MyAppConfiguration --tags key1=value1 key2=value2
+    """
+
+helps['appconfig list'] = """
+    type: command
+    short-summary: Lists all App Configurations under the current subscription.
+    examples:
+        - name: List all App Configurations under a resource group
+          text:
+            az appconfig list -g MyResourceGroup
+    """
+
+helps['appconfig show'] = """
+    type: command
+    short-summary: Show properties of an App Configuration.
+    examples:
+        - name: Show properties of an App Configuration
+          text:
+            az appconfig show -g MyResourceGroup -n MyAppConfiguration
+    """
+
+helps['appconfig credential'] = """
+    type: group
+    short-summary: Manage credentials for App Configurations
+    """
+
+helps['appconfig credential list'] = """
+    type: command
+    short-summary: List access keys of an App Configuration.
+    examples:
+        - name: List access keys of an App Configuration
+          text:
+            az appconfig credential list -g MyResourceGroup -n MyAppConfiguration
+    """
+
+helps['appconfig credential regenerate'] = """
+    type: command
+    short-summary: Regenerate an access key for an App Configuration.
+    examples:
+        - name: Regenerate a read only access key for an App Configuration
+          text:
+            az appconfig credential regenerate -g MyResourceGroup -n MyAppConfiguration --id ReadOnlyKeyId
```

Comment:
What is the actual format for `--id`?  Is it a guid?  If so, some kind of dummy guid would be better here than "ReadOnlyKeyId"

Reply:
It is not a guid, but a string generated by our RP.  One example "0-l1-s0:B74joKwEhkXsOvEDA385".